docs/book/src/roadmap.md

```diff
@@ -1,36 +1,19 @@
 # Cluster API Azure Roadmap
 
-This roadmap is a constant work in progress, subject to frequent revision. Dates are approximations. Features are listed in no particular order.
+The best place to see what's coming is in [the public milestones](https://github.com/kubernetes-sigs/cluster-api-provider-azure/milestones).  
+The next numbered milestone (i.e. **1.8**) is planned at the very begining of the 2 month release cycle and most actively planned and discussed on the community call after the major release.
+Active community PR contributions are prioritized throughout the release and unplanned work may arise, hence the items in the milestone are a rough estimate which may change.
+The "next" milestone is where items for the milestone after the current numbered one are placed to help with a very rough bucket to help prioritize upcoming work.
 
-## v0.5 (v1alpha4) ~ Q1 2021
+## High Level Vision
 
-|Area|Description|Issue/Proposal|
-|--|--|--|
-|OS|Windows worker nodes|[#153](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/153)|
-|Identity|Multi-tenancy within one manager instance|[#586](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/586)|
-|UX|Bootstrap failure detection|[#603](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/603)|
-|UX|Add tracing and metrics|[#311](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/311)|
+CAPZ is the core production-ready code to deploy self-managed or managed kubernetes clusters on Azure at scale by combining with GitOps concepts and surrounding tooling ecosystem.
```

Comment:
I would emphasize managing lifecycle of clusters over deploy, and that CAPZ is the official Azure Cluster API provider